src/xenia/cpu/xex_module.cc

```diff
@@ -82,6 +95,101 @@ const xex2_security_info* XexModule::GetSecurityInfo(
                                                      header->security_offset);
 }
 
+const PESection* XexModule::GetPESection(const char* name) {
+  for (std::vector<PESection>::iterator it = pe_sections_.begin();
+       it != pe_sections_.end(); ++it) {
+    if (!strcmp(it->name, name)) {
+      return &(*it);
+    }
+  }
+  return nullptr;
+}
+
+typedef struct mspack_memory_file_t {
```

Comment:
Could possibly move this MSPack handling code into a util library, but not a biggie at this point.

Reply:
Yeah would be nice, this is just copied from the top of xex2.cc, I'm not really sure what the best way to move it is though..